### PR TITLE
Neuvector automation for perftest

### PR DESCRIPTION
**Pull request by luigibk**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/neuvector/patches/perftest/neuvector.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+spec:
+  releaseName: neuvector
+  timeout: 600
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: neuvector-azure-keyvault
+    version: 0.0.5
+  values:
+    keyvault:
+      name: cftapps-stg
```
Comment (adusumillipraveen) [inline k8s/neuvector/patches/perftest/neuvector.yaml]:
```suggestion
      name: cftapps-test
```

File: k8s/neuvector/patches/perftest/neuvector.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      resourcegroup: core-infra-stg-rg
```
Comment (adusumillipraveen) [inline k8s/neuvector/patches/perftest/neuvector.yaml]:
```suggestion
      resourcegroup: core-infra-test-rg
```

File: k8s/neuvector/patches/perftest/neuvector.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      subscriptionid: 96c274ce-846d-4e48-89a7-d528432298a7
```
Comment (adusumillipraveen) [inline k8s/neuvector/patches/perftest/neuvector.yaml]:
```suggestion
      subscriptionid: 8a07fdcd-6abd-48b3-ad88-ff737a4b9e3c
```

**Comment by timja** [comment]
Looks to be a few outstanding changes here @luigibk ?

**Comment by luigibk** [comment]
> Looks to be a few outstanding changes here @luigibk ?

...and the secrets are missing ... don't merge it yet! :-)

**Pull request merged**